### PR TITLE
Update apache2.conf.erb

### PR DESCRIPTION
**Pull request by F0RMaTC**
Making template working (CentOS 6.5)

### CODE REVIEW

**Comment by sbotman** [comment]
Hey Bjorn,

Could you rebase this for me, so travis can rerun and I can merge your changes?
Tnx amigo.....

Sander...

File: templates/default/apache2.conf.erb [hunk 1]
```diff
@@ -11,19 +16,31 @@
...
-  ScriptAlias <%= node['nagios']['cgi-path'] %> <%= node['nagios']['cgi-bin'] %>
-  Alias /<%= node['nagios']['server']['vname'] %> <%= node['nagios']['docroot'] %>
+  ScriptAlias /<%= node['nagios']['server']['service_name'] %><%= node['nagios']['cgi-path'] %> <%= node['nagios']['cgi-bin'] %>
 
   <Directory "<%= node['nagios']['cgi-bin'] %>">
-     Options ExecCGI
+    Options ExecCGI
+    AllowOverride None
+    Order allow,deny
+    Allow from all
```
Comment (tas50) [inline templates/default/apache2.conf.erb:31]: This won't work with Apache 2.4 in RHEL 7 / Ubuntu 14.04:

http://httpd.apache.org/docs/2.4/upgrading.html

**Comment by sbotman** [comment]
sorry Bjorn, need to close this one (and I know you are not using it anymore) :)

**Comment by F0RMaTC** [comment]
Ok ;)

On May 1, 2015, at 09:54, Sander Botman <notifications@github.com<mailto:notifications@github.com>> wrote:

Closed #329https://github.com/tas50/nagios/pull/329.

##

**Pull request closed without merging**